Color changes to $communication-primary, shade-10, shade-20, shade-30

cheade opened the pull request:
#### Description of changes

The original color for $communication-primary was #0078D4, which failed color contrast when sitting on backgrounds darker than #FFFFFF. Updated our primary communication shade to #106EBE, and adjusted the darker shades to stay in line with the shading gradient. Adjusted a couple of places where elements were listed with #communiation-shade-10 as a temporary fix to the contrast issue, but should be $communication-primary. 

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #601 
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

Shobhit1 commented:
Looks like there are some changes that involve changing colors of elements in our application. If possible, can you please add some screenshot of what changed?

cheade commented:
Anything that should be using communication-primary (primary buttons, links, active command buttons). There were some items that had $communication-shade-10 as the listed color to temporarily fix the contrast issue. I updated the root $communication-primary to the darker shade and adjusted the spectrum as such. 

![image](https://user-images.githubusercontent.com/40274682/60546625-8e8a3c00-9cd2-11e9-8c5f-10d046579724.png)

![image](https://user-images.githubusercontent.com/40274682/60546705-bbd6ea00-9cd2-11e9-8d8f-5ac18fa8512a.png)

![image](https://user-images.githubusercontent.com/40274682/60546757-d6a95e80-9cd2-11e9-9911-a8de64fb7e78.png)
